prow/plugins/bugzilla/bugzilla_test.go

```diff
@@ -119,13 +119,21 @@ orgs:
 <li>on the "my-repo-branch" branch, valid bugs must be closed, target the "my-repo-branch" release, and be in one of the following states: MODIFIED. After being linked to a pull request, bugs will be moved to the PRE state, updated to refer to the pull request using the external bug tracker, and moved to the MODIFIED state when all linked pull requests are merged.</li>
 </ul>`,
 		},
-		Commands: []pluginhelp.Command{{
-			Usage:       "/bugzilla refresh",
-			Description: "Check Bugzilla for a valid bug referenced in the PR title",
-			Featured:    false,
-			WhoCanUse:   "Anyone",
-			Examples:    []string{"/bugzilla refresh"},
-		}},
+		Commands: []pluginhelp.Command{
+			{
+				Usage:       "/bugzilla refresh",
+				Description: "Check Bugzilla for a valid bug referenced in the PR title",
+				Featured:    false,
+				WhoCanUse:   "Anyone",
+				Examples:    []string{"/bugzilla refresh"},
+			}, {
+				Usage:       "/bugzilla qa",
+				Description: "Assign PR to QA contact specified in Bugzilla",
+				Featured:    false,
+				WhoCanUse:   "Anyone",
+				Examples:    []string{"/bugzilla qa"},
```

Comment:
This needs to reflect the change of command to `assign-qa` too (both usage and examples).